bindings/rust/src/lib.rs

```diff
@@ -42,7 +42,19 @@ use std::ptr::NonNull;
 
 /// Parse and validate the input according to WebAssembly 1.0 rules. Returns true if the supplied input is valid.
 pub fn validate<T: AsRef<[u8]>>(input: T) -> bool {
-    unsafe { sys::fizzy_validate(input.as_ref().as_ptr(), input.as_ref().len()) }
+    let mut err = Box::new(sys::FizzyError {
+        code: 0,
+        message: [0i8; 256],
+    });
+    let ret =
+        unsafe { sys::fizzy_validate(input.as_ref().as_ptr(), input.as_ref().len(), &mut *err) };
```

Comment:
Apparently `&mut *err` is the suggested option: https://doc.rust-lang.org/std/primitive.pointer.html#1-coerce-a-reference-t-or-mutable-reference-mut-t